app/chitter.rb

```diff
@@ -0,0 +1,83 @@
+require 'sinatra/base'
+require 'sinatra/flash'
+require_relative 'data_mapper_setup'
+require './app/helpers/current_user'
+
+
+class Chitter < Sinatra::Base
+
+  include CurrentUser
+
+  enable :sessions
+  register Sinatra::Flash
+  set :session_secret, 'super secret'
+
+  set :static, true
+  set :root, File.dirname(__FILE__)
+  set :public_folder, 'public'
+  set :views, proc {File.join(root, '..', 'app/views')}
+
+  get '/' do
+    @peeps = Peep.all.reverse
+    erb(:'index')
+  end
+
+  get '/users/sign_up' do
+    erb(:"users/sign_up")
+  end
+
+  post '/users/sign_up' do
+    if params[:password] != params[:password_confirmation]
+      flash.now[:errors] = []
+      flash.now[:errors] << "Your passwords didn't match. Try again."
+      erb(:"users/sign_up")
+    else
+      user = User.new(name: params[:name], username: params[:username], email: params[:email], password: params[:password])
```

Comment:
Metrics/LineLength: Line is too long. [123/90] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)